modules/raft/src/main/java/org/apache/ignite/internal/raft/RaftGroupServiceImpl.java

```diff
@@ -314,21 +314,29 @@ public CompletableFuture<Void> removePeer(Peer peer) {
     }
 
     @Override
-    public CompletableFuture<Void> changePeers(Collection<Peer> peers) {
+    public CompletableFuture<Void> changePeers(PeersAndLearners peersAndLearners, long term) {
```

Comment:
So, changePeers now change peers and learners actually, it's confusing a little bit

Reply:
Yep, just like changePeersAsync that also changes both peers and learners. Do you believe that I should rename both?

Reply:
After discussion - changed to changePeersAndLearners, thanks!